doc/man3/PKCS12_add_friendlyname_asc.pod

```diff
@@ -0,0 +1,45 @@
+=pod
+
+=head1 NAME
+
+PKCS12_add_friendlyname_asc, PKCS12_add_friendlyname_utf8, PKCS12_add_friendlyname_uni - Functions to add the friendlyname attribute to a PKCS#12 safeBag
+
+=head1 SYNOPSIS
+
+ #include <openssl/pkcs12.h>
+
+ int PKCS12_add_friendlyname_asc(PKCS12_SAFEBAG *bag, const char *name,
+                                int namelen);
+
+ int PKCS12_add_friendlyname_utf8(PKCS12_SAFEBAG *bag, const char *name,
+                                int namelen);
+
+ int PKCS12_add_friendlyname_uni(PKCS12_SAFEBAG *bag,
+                                const unsigned char *name, int namelen);
```

Comment:
Nit: looks like there is an indentation issue here. The params on each of the second lines above should line up with the start of the params list on the line before.